### PR TITLE
[Doc] Fix docs deploy netlify test configuration

### PR DESCRIPTION
**Pull request by Justinterest**
Related issue = #

### CODE REVIEW

**Review by rexwangcc — COMMENTED** [review]
I left one comment otherwise LGTM!

File: netlify.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
+  command = "git clone https://github.com/taichi-dev/docs.taichi.graphics.git; git clone https://github.com/taichi-dev/docstring-gen docsgen; export DOCSTRING_GEN_PATH=\"$(pwd)/docsgen\"; export TAICHI_PATH=\"$(pwd)/python/taichi\"; export TAICHI_MAIN=\"$(pwd)\"; export TAICHI_WEBSITE=\"$(pwd)/docs.taichi.graphics\"; pip install sphinx-autoapi==1.8.4 gitpython pydata-sphinx-theme==0.7.2; cd $DOCSTRING_GEN_PATH/experimental; export current_version=master; make clean; make version; make apideploy; cd $TAICHI_WEBSITE/website; npm install --global yarn@1.22; yarn install; ./sync_docs.sh dev; yarn build;"
```
[the remark this answers is not in the document]
Comment (Justinterest) [reply netlify.toml:2]: `./sync_docs.sh dev` will do sync the latest master branch content from `taichi-dev/taichi`

[the remark this answers is not in the document]
Comment (rexwangcc) [reply netlify.toml:2]: Gotcha, thanks!

**Review by rexwangcc — APPROVED** [review]

File: netlify.toml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Justinterest) [reply netlify.toml:2]: And add diff check for netlify.toml
From the log output, can found the sync log
image

**Pull request merged**